apps/codeforafrica/src/pages/admin/UUIDWidget.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```

Comment:
We don't need this file.

Reply:
I thought having all custom widgets(registration) under one file is better than always adding them under the index page every time we need a new widget.